deploy/openj9-chart/values.yaml

```diff
@@ -0,0 +1,39 @@
+###############################################################################
+# Copyright (c) 2019,2020 IBM Corp.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###############################################################################
+
+fullnameOverride: []
+
+arch: amd64
+
+image:
+  repository: docker.io/adoptopenjdk
+  tag: openj9
+  pullPolicy: Always
+
+container:
+  command: ["jitserver"]
+  OPENJ9_JAVA_OPTIONS: "-Xjit:verbose"
```

Comment:
I think we need to remove verbose and use it only for debug/test.

Reply:
I see, `-Xjit:verbose` has been removed.